ui/src/app/components/runs/run-detail/run-detail.component.html

```diff
@@ -13,9 +13,7 @@
   ~ limitations under the License.
   -->
 
-<div class="content-area">
-  <clr-datagrid (clrDgRefresh)="refresh($event)" [clrDgLoading]="loading">
-    <clr-dg-column>ID</clr-dg-column>
+  <clr-datagrid [clrDgLoading]="loading">
     <clr-dg-column>Name</clr-dg-column>
```

Comment:
I would rename the column header to `Job Name`

Reply:
Done